ui/app/styles/core/element-styling.scss

```diff
@@ -103,32 +103,38 @@ pre {
 }
 
 body {
-  font-size: 1rem;
+  font-size: $size-6;
   line-height: 1.5;
 }
 
 a {
-  color: #1563ff;
+  color: $blue-500;
   cursor: pointer;
   text-decoration: none;
+
+  &:hover,
+  &:active,
+  &:focus {
+    position: relative;
+  }
 }
 a strong {
   color: currentColor;
 }
 a:hover {
-  color: #1f2124;
+  color: $ui-gray-900;
 }
 
 code {
   background-color: transparent;
-  color: #525761;
-  font-size: 0.875em;
+  color: $ui-gray-700;
+  font-size: 1em;
   font-weight: normal;
-  padding: 0.25em 0.5em 0.25em;
+  padding: 0;
```

Comment:
padding: 0 came from the generic file and thus was overriding the bulma elements. I confirmed this by looking at the binary css.

Reply:
same with font size.